pkg/providers/pricing/metrics.go

```diff
@@ -0,0 +1,57 @@
+/*
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package pricing
+
+import (
+	"strings"
+
+	"github.com/prometheus/client_golang/prometheus"
+	crmetrics "sigs.k8s.io/controller-runtime/pkg/metrics"
+
+	"github.com/aws/karpenter-core/pkg/apis/v1alpha5"
+	"github.com/aws/karpenter-core/pkg/metrics"
+	v1 "k8s.io/api/core/v1"
+)
+
+const (
+	pricingSubsystem = "pricing"
+)
+
+var (
+	instanceEstimate = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Namespace: metrics.Namespace,
+			Subsystem: pricingSubsystem,
+			Name:      "instance_estimate",
+			Help:      "Estimated hourly price used when making informed decisions on node cost calculation. This is updated once on startup and then every 12 hours.",
+		},
+		[]string{
```

Comment:
@bwagner5 I would like to add in `vcpu` and `memory` labels here as well so we can have metrics that compute CPU and memory pricing, but the `instancetype` provider depends on `pricing` currently. 

Any recommendations? I thought about extracting the cache out from `instancetype` provider if that makes sense.

Reply:
Are you able to pull this out into the instancetype.go provider? Seems like `createOfferings()` has all the information you need to label this information.

Reply:
That sounds reasonable!